.github/workflows/ci-master.yml

```diff
@@ -33,13 +33,9 @@ jobs:
         run: pnpm build
 
       - name: Cache build artifacts
-        uses: actions/cache@v3
+        uses: actions/cache/save@v3.3.1
         with:
-          path: |
-            /github/home/.cache
-            /github/home/.pnpm-store
-            ./node_modules
-            ./packages
```

Comment:
Is there a way to monitor GH Actions cache so we can see this coming next time? Or by trimming down cache size this shouldn't be necessary?